Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -115,9 +115,19 @@ - (void)initializeWithParameters:(id<MAAdapterInitializationParameters>)paramete
     
     if ( [ALVungleInitialized compareAndSet: NO update: YES] )
     {
+        [self initVungleSDK:parameters completionHandler:completionHandler];
+    }
+    else
+    {
+        completionHandler(ALVungleIntializationStatus, nil);
+    }
+}
+
+- (void)initVungleSDK:(id<MAAdapterInitializationParameters>)parameters completionHandler:(void (^)(MAAdapterInitializationStatus, NSString *_Nullable))completionHandler {
```

Comment:
This is different from the Android and AdMob iOS implementation. If the ads call this and it's ongoing, it will immediately fail instead of queuing up and waiting.

Reply:
Is the difference expected?